### PR TITLE
Merge sync-build branch

### PR DESCRIPTION
**Pull request by vidurb**
Contains the addition of the packages and configuration required for building assets.

### CODE REVIEW

File: assets/paths.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      ],
+      watch_config: {
+        ignored: '../templates/_manifest.json',
+      },
+    },
+  },
+  js: {
+    urgent: {
+      source: [
+      ],
```
Comment (rungta) [inline assets/paths.js:67]: Add the font loader files and lazysizes files to the `urgent` task.

File: assets/paths.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      destination: 'urgent.js',
+      watch: [
+      ],
+    },
+    deferred: {
+      source: [
+      ],
```
Comment (rungta) [inline assets/paths.js:78]: Add the PJAX files to the `deferred` task.

**Review by rungta — CHANGES_REQUESTED** [review]

File: assets/paths.js (hunk 1 above)
Comment (vidurb) [reply assets/paths.js:67]: Added. Note that IMHO this should have been on the sync-js branch.

File: assets/paths.js (hunk 2 above)
Comment (vidurb) [reply assets/paths.js:78]: Added. Note that IMHO this should have been on the sync-js branch.

File: assets/paths.js (hunk 1 above)
Comment (rungta) [reply assets/paths.js:67]: Ah yes. My bad.

**Pull request merged**